tests/test_tracing_tracer.py

```diff
@@ -334,26 +337,26 @@ def test_threaded_memory(tracer, asserts):
 
 
 @pytest.mark.parametrize('google_headers,expected_headers', [
-    ({"X-Cloud-Trace-Context": "trace-id/span-id;options"},
+    ({"X-Cloud-Trace-Context": f"{TEST_32_CHAR_TRACE_ID}/{TEST_16_CHAR_SPAN_ID};options"},
      {
-         "X-B3-SpanId": "span-id",
-         "X-B3-TraceId": "trace-id"
+         "X-B3-SpanId": TEST_16_CHAR_SPAN_ID,
+         "X-B3-TraceId": TEST_32_CHAR_TRACE_ID
      }),
-    ({"X-Cloud-Trace-Context": "trace-id/span-id;"},
+    ({"X-Cloud-Trace-Context": f"{TEST_32_CHAR_TRACE_ID}/{TEST_16_CHAR_SPAN_ID};"},
      {
-         "X-B3-SpanId": "span-id",
-         "X-B3-TraceId": "trace-id"
+         "X-B3-SpanId": TEST_16_CHAR_SPAN_ID,
+         "X-B3-TraceId": TEST_32_CHAR_TRACE_ID
      }),
-    ({"X-Cloud-Trace-Context": "trace-id/span-id"},
+    ({"X-Cloud-Trace-Context": f"{TEST_32_CHAR_TRACE_ID}/{TEST_16_CHAR_SPAN_ID};options"},
      {
-         "X-B3-SpanId": "span-id",
-         "X-B3-TraceId": "trace-id"
+         "X-B3-SpanId": TEST_16_CHAR_SPAN_ID,
+         "X-B3-TraceId": TEST_32_CHAR_TRACE_ID
      }),
     ({"X-Cloud-Trace-Context": "malformed"},
-     {}
-     ),
+     {}),
+    ({"X-Cloud-Trace-Context": "short-trace-id/short-span-id;options"},
```

Comment:
It might make the test simpler (and therefore help with the maintanaince) to have it in a separate function with the use case in the test name (eg `test_short_trace_id`).

Reply:
I did have this originally (see this commit https://github.com/bbc/logtracer/pull/6/commits/dcc834e49816873fcb617d4e91ea12402adfa13f) but @jensraaby suggest I parametrise them https://github.com/bbc/logtracer/pull/6#discussion_r245016501 - maybe there's a better way I could have done this?

Reply:
Yeah I wasn't sure which would be easier to read. If the parameters are compact I think parameterised tests can read well. It might help to have a descriptive parameter as well as the others in this case if it's not obvious what's being tested. 
The other test cases were getting quite big so the alternative was to make them smaller and have more with descriptive names like you're suggesting @tatiana .